coalib/settings/ConfigurationGathering.py

```diff
@@ -394,13 +394,14 @@ def gather_configuration(acquire_settings,
         arg_list = sys.argv[1:] if arg_list is None else arg_list
     sections, targets = load_configuration(arg_list, log_printer, arg_parser,
                                            args=args)
-    local_bears, global_bears = fill_settings(sections,
-                                              acquire_settings,
-                                              log_printer)
+    local_bears, global_bears, aspects = fill_settings(sections,
+                                                       acquire_settings,
+                                                       log_printer)
     save_sections(sections)
     warn_nonexistent_targets(targets, sections, log_printer)
 
     return (sections,
             local_bears,
             global_bears,
+            aspects,
```

Comment:
IDK it wiill be bad :(

This is required for bear collection method and processing.py to have access to the same AspectList instance, but It's not mandatory. I can initialize AspectList 2 times  from `section` and deleting the `section['aspects']` in the last usage.